spec/support/acceptance/splunk/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "2.0"
+
+services:
+  enterprise:
+    image: splunk/splunk:latest
+    hostname: splunk_instance
+    environment:
+      - SPLUNK_START_ARGS=--accept-license
+      - ASFDGASDFG=5
+      # The splunkbase credentials are to download and install the Puppet
+      # Report Viewer from Splunkbase.
+      # We could alternatively download the packaged app from a location
+      # like github for testing.
+      - SPLUNK_APPS_URL=https://splunkbase.splunk.com/app/4413/release/3.0.1/download
```

Comment:
This is a persistent instance right?

Reply:
Yeah this is just a splunkbase account that is used to download splunk apps. It's separate from a login to a splunk instance.

Reply:
I tried to create a team account, but team-pie@puppet.com already has an account. I'll update it as soon as I find the password haha.